picky/Cargo.toml

```diff
@@ -86,3 +86,4 @@ ssh = ["byteorder", "aes", "ctr", "cbc", "bcrypt-pbkdf", "lexical-sort"]
 http_trait_impl = ["http"]
 chrono_conversion = ["chrono", "picky-asn1/chrono_conversion"]
 time_conversion = ["time", "picky-asn1/time_conversion"]
+zeroize = ["picky-asn1-x509/zeroize", "picky-asn1/zeroize"]
```

Comment:
I introduced it as a feature because of the behavior, not for the deps. I'll use picky exclusively with this features enabled so it's not a big deal for me to put it as a mandatory behavior. Tell me if you want zeroize in the default features o just use zeroize as mandatory.

Reply:
Yes, I think it's good to have it mandatory